run_editor.sh

```diff
@@ -15,5 +15,13 @@ TB3_UE_DIR=${1:-"${CURRENT_DIR}"}
 # Configure environment for FastDDS discovery
 source ${TB3_UE_DIR}/fastdds_setup.sh
 
+#change default level, generating DefautlEngine.ini
+DEFAULT_LEVEL=${LEVEL_NAME:-"Turtlebot3_benchmark"}
+DEFAULT_RATE=${FIXED_FRAME_RATE:-"100.0"}
+DEFAULT_RTF=${TARGET_RTF:-"1.0"}
+sed -e 's/${LEVEL_NAME}/'${DEFAULT_LEVEL}'/g' Config/DefaultEngineBase.ini > Config/DefaultEngine.ini
+sed -i -e 's/${FIXED_FRAME_RATE}/'${DEFAULT_RATE}'/g' Config/DefaultEngine.ini
+sed -i -e 's/${TARGET_RTF}/'${DEFAULT_RTF}'/g' Config/DefaultEngine.ini
+
```

Comment:
It looks like both `DefaultEngine.ini & DefaultEngineBase.ini` are added to git at the moment, so should `DefaultEngine.ini` be removed from git tracking & added to `.gitignore`?